raft/src/test/java/org/apache/kafka/raft/LeaderStateTest.java

```diff
@@ -44,25 +48,26 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class LeaderStateTest {
-    private final int localId = 0;
-    private final int epoch = 5;
+    private final Entry<Integer, VoterSet.VoterNode> localId = entry(0);
+    private final Entry<Integer, VoterSet.VoterNode> epoch = entry(5);
```

Comment:
1. It's not common to create an entry object
2. I don't think directryId has any effect in most of these test cases, so could we use a dummy one to represent it?

Reply:
Any update to this comment?

Reply:
I will fix it a little bit later. Now struggling with the `DescribeQuorumRequestTes`.

Reply:
Fixed.